Gulpfile.coffee

```diff
@@ -92,6 +92,12 @@ gulp.task 'tdd', ['build'],  (done) ->
   return # Since this is async
 
 gulp.task 'buildJS', ['cleanJS'], -> build(false)
+gulp.task 'watchJS', ['cleanJS'], (done)->
+  build(true)
+  buildTests(true).on 'end', ->
+    config =
+      configFile: __dirname + '/test/karma.config.coffee'
+    karma.server.start(config, done)
```

Comment:
This is now in a separate function.  The singleRun option in karma would stop if it was false.  Now that is being accounted for.